src/utils/string/StringUtils.js

```diff
@@ -248,7 +249,7 @@ const styles = StyleSheet.create({
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 17,
-    color: '#000000'
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from hard-coded color value to using a constant from the COLORS object is a good practice for maintainability and consistency. However, please ensure that `COLORS.black` is indeed equivalent to `'#000000'`.
```diff
-    color: '#000000'
+    color: COLORS.black
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->